src/configuration.h

```diff
@@ -128,6 +128,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LPS22HB_ADDR_ALT 0x5D
 #define SHT31_4x_ADDR 0x44
 #define PMSA0031_ADDR 0x12
+#define AHT10_ADDR 0x38
```

Comment:
NCP5623 uses this address already. Please add scanner code to detemine which device is actually connected

Reply:
NCP5623 and AHT10 does not have a simple method to check their identity, so if NCP5623 is defined (present in specific cards), no AHT10 detection will be performed.